recipes/matrix_client/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "matrix_client" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: dce3ccb8665df0d519f08e07a16e6d3f9fab3a947df4b7a7c4bb26573d68f2d5
+
+build:
+  number: 0
+  # pytest-runner 5.1 doesn't exist for py38.
+  skip: true  # [py38]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pytest-runner 5.1
```

Do you really need 5.1 exactly?
```suggestion
    - pytest-runner >=5.1
```

---

Good point, I modified it and it passes tests locally.

I had kept it same to match what was in source, given my tests were failing earlier but that was when I was testing for python 3.8.